RELEASING.md

```diff
@@ -83,16 +93,29 @@ Before attempting to create a Calico release you must do the following.
    - [calico/typha](https://github.com/projectcalico/typha/releases)
    - [calico/dikastes](https://github.com/projectcalico/app-policy/releases)
    - [calico/pod2daemon-flexvol](https://github.com/projectcalico/pod2daemon/releases)
-
-   The following components _must_ use the same minor revision number as the Calico version number above, but
-   do not always need to be released for every patch. They must be cut as part of a minor release of Calico.
-
    - [networking-calico](https://github.com/openstack/networking-calico)
```

Comment:
This has a "Releasing debs and rpms" (https://github.com/openstack/networking-calico/blob/master/RELEASING.md#releasing-debs-and-rpms), should that be updated to remove that? I think it would also be good to add a note or something here that only the release in the repo should be done as the packages will be built and published in the next section.

Reply:
Yes, I'm working on that for networking-calico at https://review.opendev.org/#/c/674959/, and for Felix at https://github.com/projectcalico/felix/pull/2100.  In both cases the RELEASING.md will now say:

# Debian and RPM packages

For the preparation of Debian and RPM packages, see
https://github.com/projectcalico/packaging.

(Tigera builds and publishes packages for each Calico release as a
whole, once all the Calico components have been released.)

How does that sound?